src/Sql/Sql/help/Set-AzSqlInstanceLink.md

```diff
@@ -0,0 +1,352 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version:
+schema: 2.0.0
+---
+
+# Set-AzSqlInstanceLink
```

Comment:
Could you check the API method of `Set-AzSqlInstanceLink` is a patch/put operation? If it is a patch operation, please use Update-AzSqlInstanceLink to rename this parameter. Please check Set-Az*Certificates as well.

Reply:
I believe it's `patch`. I'll just need to confirm with PMs that this change is ok (since we already had Set-* approved)
Az*Certificates doesn't have Set/Update operation for now so we're good there

Reply:
Might have to push back on this one as it seems like we have already partially released the documentation with Set* cmdlet.
Is it okay if we keep it as `Set-AzSqlInstanceLink` @BethanyZhou ?

cc: @danimir @sasapopo

Reply:
According to our [cmdlet best practice](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md?plain=1), there should be Update-. Considering current situation where your team partially released the documentation with Set* cmdlet. I think we can keep Update-* as official name and set `Set-*` as alias of this cmdlet. What's your thought?

Reply:
Sure. It's okay to keep them both for a long time.